website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
```

Comment:
Oh, I had copied that from [espree](https://github.com/fkling/astexplorer/blob/93aa4be41e00e0754afc97c0f9eebd26d401eaca/website/src/parsers/js/espree.js#L13). Should we change it there too? It looks like [esprima](https://github.com/fkling/astexplorer/blob/93aa4be41e00e0754afc97c0f9eebd26d401eaca/website/src/parsers/js/esprima.js#L14) and [recast] already (https://github.com/fkling/astexplorer/blob/93aa4be41e00e0754afc97c0f9eebd26d401eaca/website/src/parsers/js/recast.js#L11) default to `false`.